.github/workflows/release.yaml

```
@@ -222,8 +245,25 @@ jobs:
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - name: Kubernetes ${{ matrix.kubernetes-version }} E2E Tests
-        run: KONG_CLUSTER_VERSION=${{ matrix.kubernetes-version }} make test.e2e
+      - uses: Kong/kong-license@master
+        id: license
+        with:
+          password: ${{ secrets.PULP_PASSWORD }}
+      - name: Parse semver string
+        id: semver_parser
+        uses: booxmedialtd/ws-action-parse-semver@v1
+        with:
+          input_string: ${{ github.event.inputs.tag }}
+          version_extractor_regex: 'v(.*)$'
+      - name: run ${{ matrix.test }} - ${{ matrix.kubernetes-version }}
```

```suggestion
      - name: ${{ matrix.test }} - ${{ matrix.kubernetes-version }}
```

The shorter the name the more of it one can see in Github's workflows' view.